spec/features/workflow/show_with_creator_spec.rb

```diff
@@ -10,15 +10,16 @@
 
   def given_there_is_a_document_with_a_creator
     @user = create(:user, name: "Joe Bloggs")
```

Comment:
Could we use `current_user` here instead, just to avoid the coupling with the search filters?

Reply:
Sure